packages/taquito/src/wallet/wallet.ts

```diff
@@ -408,8 +415,4 @@ export class Wallet {
     );
     return contractAbstractionComposer(abs, this.context);
   }
-
-  getPK() {
```

Comment:
@dsawali  I removed `getPK` function cuz it's name and logic are consistent with `class Wallet` function `pkh` and add `pk` at line 213 instead. Second thought now, i wasn't sure if any user already use this `getPK` function that this will be breaking people's code. Do you think we should add getPK back and call `pk` underneath and mark it as deprecated before officially removed it?